### ISSUE TITLE
[SCREENREADER]: GIBCT EYB - Live region might start announcing in IE11 + JAWS with aria-relevant added

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16002]
## [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a JAWS + IE11 user, I want to hear my updated benefits read aloud when I make a change on the GI Bill Comparison Tool profile page.

## Details

Our live region for benefits isn't announcing the change in IE11 + JAWS. This may be an implementation feature, that it's not going to announce itself unless the container is emptied and HTML re-added, or it **could** be because we're not listening for live-region changes, just additions. Please see this WebAIM thread https://webaim.org/discussion/mail_thread?thread=9398 from last year discussing this exact thing. Screenshot and code snippet to experiment with attached below.

## Acceptance Criteria

- [ ] Live region announces changes in IE11 + JAWS when users update one or more fields in the Estimate Your Benefits section
- [ ] No axe violations in Chrome/Firefox dev tools

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* Win10
* IE11
* JAWS 2019/2020
* https://staging.va.gov/gi-bill-comparison-tool/profile/11806124

## Solution (if known)

This may not work, but it might be exactly what we need.

```diff
<div
  aria-atomic="true"
  aria-live="polite"
+ aria-relevant="additions text"
  role="status"
>
  ...
</div>
```
-->

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-09 at 10 04 58 PM](https://user-images.githubusercontent.com/934879/98627132-1590f080-22d9-11eb-8534-1ed7aa9e1908.png)

**Comment by ksantiagomorgan6** [comment]
@1Copenut I tried your suggestion and it did not work. This is related to this bug https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/11300. Looks like someone from freedom scientific might of picked up the issue on September 2nd.

**Comment by 1Copenut** [comment]
Thank you @ksantiagomorgan6 I hadn't seen that it was picked up. I guess we can hold on this one for now as a known upstream issue. I'll put a watch on the issue and check back periodically.

**Comment by 1Copenut** [comment]
@desireeturner https://github.com/FreedomScientific/VFO-standards-support/issues/408 is believed to be the root cause of this issue. Chrome and Edge Chromium correctly announce the updated change, so I'm moving this one to close with the label `wontfix`.
